api/Document.json

```diff
@@ -4413,12 +4426,11 @@
                 ]
               }
             ],
-            "chrome_android": "mirror",
             "edge": {
               "version_added": "85"
             },
             "firefox": {
-              "version_added": "65"
+              "version_added": "117"
```

Comment:
I see that Firefox is getting updated as well, is that intentional?  If so, could you provide supporting documentation for these changes?